test/index.ts

```diff
@@ -108,4 +108,12 @@ describe('decodeResumeToken', function() {
       documentKey: null
     });
   });
+
+  // See PM-1950
+  it.only('can decode new resume tokens', async() => {
+    const decoded = decodeResumeToken('826273BCD2000000012B022C0100296E5A1004808BBE9298B0403286C29EA48C4E162846645F696400646273BCD27257D30393BB73F70004');
```

Comment:
I got this from a new change stream event on 6.0-rc0. I can't find any tests of the new format or specs or anything anywhere yet. Maybe I'm not very good at looking.